src/2048-expert/index.js

```diff
@@ -178,13 +181,14 @@ function mergeVertical() {
             }
         }
     }
+    checkWin()
 }
 
 
 // update the current score
```

Comment:
the best score is also being updated here so maybe `update the current and best score` would be a better function description